docs/description/js_tour.rst

```diff
@@ -36,22 +32,22 @@ Example::
                 element:   '.oe_mail_wall .oe_msg.oe_msg_composer_compact>div>.oe_compose_post',
             },
         ]
-    });
-    }());
 
 What you do here is describing steps that got to be proceeded by user or phantom (phantomjs).
 
 Important details:
 
     * **id** - need to call this tour
-    * **path** - from this path tour will be started
+    * **path** - from this path tour will be started. Used in test mode only!
```

Comment:
"Used in test mode only!" was a message for you @ilyasProgrammer . It was not a quote to add to file